akka-docs/src/main/paradox/stream/operators/Source-or-Flow/zipAll.md

```diff
@@ -16,6 +16,13 @@ Combines all elements from each of multiple sources into @scala[tuples] @java[*P
 
 Combines all elements from each of multiple sources into @scala[tuples] @java[*Pair*] and passes the @scala[tuples] @java[pairs] downstream.
 
+See also:
+
+ * @ref:[zip](zip.md)
+ * @ref:[zipWith](zipWith.md)
+ * @ref:[zipWith](zipWith.md)  
+ * @ref:[zipWithIndex](zipWithIndex.md)
+
 ## Example
 Scala
 :   @@snip [FlowZipSpec.scala](/akka-stream-tests/src/test/scala/akka/stream/scaladsl/FlowZipSpec.scala) { #zip }
```

Comment:
That is a `zip` example, not `zipAll`?

Reply:
Didn't change this but I'll look into it.

Reply:
Actually, just noticed after removing that sample, the entire page is not about the `zipAll` operator but copy-pasta about `zip`...

Reply:
Added example for that as well now